provisioner/terraform/testdata/calling-module/calling-module.tfstate.json

```diff
@@ -26,7 +26,7 @@
               }
             ],
             "env": null,
-            "id": "c559c8be-3b52-444a-bf51-81d270002ec6",
+            "id": "14f0eb08-1bdb-4d48-ab20-e06584ee5b68",
```

Comment:
(not in scope of this PR): Same here, it's annoying that these get regenerated and cause diffs. Could we replace these with placeholders?